app/models/katello/host/subscription_facet.rb

```diff
@@ -34,6 +35,19 @@ def import_database_attributes(consumer_params)
         end
       end
 
+      def update_consumer_type(consumer_params)
+        if consumer_params.try(:[], 'type').try(:[], 'label') == 'hypervisor'
+          self.consumer_type = 'hypervisor'
+        elsif consumer_params.try(:[], 'facts').try(:[], 'virt.is_guest') == "true"
```

Comment:
I think i'd disagree with you @thomasmckay.  A guest is a guest, it doesn't matter if we know about its hypervisor or not?

I've heard complaints from users about not knowing if a guest's hypervisor is registered or not due to virt-who miconfiguration which makes me think they would want to consider them guests.  Why wouldn't you?